static/app/components/core/textarea/index.tsx

```diff
@@ -27,24 +29,22 @@ export interface TextAreaProps
 }
 
 const TextAreaControl = forwardRef(function TextAreaControl(
-  {autosize, rows, maxRows, size: _size, ...p}: TextAreaProps,
+  {autosize, rows = 3, maxRows, size: _size, ...p}: TextAreaProps,
   ref: React.Ref<HTMLTextAreaElement>
 ) {
   return autosize ? (
-    <TextareaAutosize {...p} ref={ref} rows={rows ? rows : 2} maxRows={maxRows} />
+    <TextareaAutosize {...p} ref={ref} rows={rows} maxRows={maxRows} />
```

Comment:
slight breaking change: For `TextareaAutosize`, we now also have 3 rows per default instead of 2. This would have an impact on the current theme, but almost all usages pass an explicit `rows` prop.